tests/test_my_registrations.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from pages.registrations import MyRegistrationsPage
+from pages.registries import RegistrationAddNewPage, RegistrationDraftPage, RegistrationDetailPage
+
+
+class TestMyRegistrationsPageEmpty:
+    """This test covers the My Registrations page: https://{home}/registries/my-registrations
+    """
+    @pytest.fixture()
+    def registrations_page(self, driver, must_be_logged_in):
+        my_registrations_page = MyRegistrationsPage(driver)
+        my_registrations_page.goto()
+        return my_registrations_page
+
+    @pytest.fixture()
+    def my_registrations(self, driver, registrations_page):
```

Comment:
looks like this fixture isn't used -- can it be removed?

Reply:
Yup